### ISSUE TITLE
Question:  Normalization required?

### ISSUE THREAD

**Issue by jmrichardson** [sktime/sktime-dl#62]
Hi, I am just taking a look at how to use sktime-dl and noticed that the[ example data](https://github.com/sktime/sktime-dl/blob/dev/examples/multivariate_time_series_classification.ipynb) already appears to be normalized/standardized.  Is this a requirement or is there a normalization layer included in the classifiers?

Not sure if this is the right forum to be asking questions.  If not, is there a better place?  TIA

**Comment by mloning** [comment]
Hi, this is the right place to ask questions! :) 

We encode the label (y) as far as I know in the model but not the time series feature data (X). And yes, the data sets from timeseriesclassification.com are all standard normalised. 

To standardise the data, you could use sktime: 

```python
from sktime.transformers.series_as_features.compose import RowTransformer
from sklearn.preprocessing import StandardScaler
from sktime.datasets import load_arrow_head

X, y = load_arrow_head(return_X_y=True)
scaler = RowTransformer(StandardScaler())  # applies StandardScaler to each row/time series
Xt = scaler.fit_transform(X)
```

**Comment by jmrichardson** [comment]
Hi @mloning ,

Thank you for your help!  Question, my understanding is it's best practice to fit the standardscaler to the training data, then use that fit to transform both training and then test versus transforming training and test at the same time.   

Performing a quick test:
```
        scaler = RowTransformer(StandardScaler())
        s = scaler.fit(self.X_train)
        Xtrain = s.transform(self.X_train)
        Xtest = s.transform(self.X_test)
        Xtest_tf = scaler.fit_transform(self.X_test)
```

Both Xtest and Xtest_tf are identical.  Am I correct in that the fit/transform is happening on each individual sequence (each time series) independently.  If so, is this not an issue (perhaps because the data is time series versus tabular)?  My concern is that when new production time series data arrives, then shouldn't we be transforming it the same way as in training (using the fitted parameters in standard scaler)?  Also, don't we lose the ability to inverse_transform the data?:

```
        scaler = RowTransformer(StandardScaler())
        Xtrain = scaler.fit_transform(self.X_train)
        Xtrain_inv = scaler.inverse_transform(Xtrain)
```

I would have thought that the above code self.X_train and Xtrain_inv would be identical but they differ significantly.  

Thanks for any insight and really enjoying sktime.  I have decided to use sktime for all my time series needs and really looking forward to some of the enhancements you smart folks are working on.

**Comment by mloning** [comment]
1) I think applying the standard scaler to each series is okay as long as we don't learn anything from the test set during fitting which we then use to generate predictions on the test (leakage). The fit and transform each individual series and then basically forget about the learnt parameters (mean and std). You could of course try to estimate a single mean and std from a training set that standardises all instances which you can then use to standardise test instances but that seems to be a different problem. 
2) You're right, `inverse_transform` seems broken, the problem is that we don't store the fitted standard scaler for each series, which we would need to properly do the inverse transform. Could you open an issue for that in sktime's repo with the same example? There is even a unit test for it [here](https://github.com/alan-turing-institute/sktime/blob/70b82aad8c0c2263fe3a9b07e0174e41c7bc3e65/sktime/transformers/series_as_features/tests/test_compose.py#L59), but it passes because we test it on already standardised data ... so that needs fixing too. Perhaps we should rewrite the RowTransformer and use a simple for-loop over instances even though that's terribly slow. We could try to speed it up with numba or joblib.

**Comment by jmrichardson** [comment]
Thank you and just submitted the bug.
